cranelift-wasm/tests/wasm_testsuite.rs

```diff
@@ -45,6 +45,28 @@ fn use_fallthrough_return() {
     );
 }
 
+#[test]
+fn use_name_section() {
+    let wat = read_file(Path::new("../wasmtests/names.wat")).expect("error reading wat file");
```

Comment:
Light suggestion, feel free to dismiss: since the names.wat test is so small, and there's now test information that's scattered (in both names.wat and here, the "$func_name" name), could we just inline the content of names.wat here instead?